public/locales/en/common.json

```diff
@@ -90,7 +90,7 @@
       "to-vault": "To vault",
       "to-wallet": "To wallet",
       "token-input": {
-        "search-select-token": "Select a Token",
+        "search-select-token": "Select a token to ZAP",
```

Comment:
The underlying token is also available to select from the token options list on transactions, so its not technically a Zap token.

Reply:
the underlying token is the default - 
so i would probably not click to view the modal to zap a token unless i want to zap